### ISSUE TITLE
GDAX support wanted

### ISSUE THREAD

**Issue by tomekwl** [butor/blackbird#281]
Browsing through the source code I noticed GDAX is barely implemented. Any plans to make it run?

**Comment by bertrandfalguiere** [comment]
Yeah, and since it's another exchange with short selling, it would give a real leverage.

**Comment by olliec** [comment]
Has anyone had much luck with this?
Maybe my volumes aren't significant enough!!

**Comment by brentonmcs** [comment]
I was having a look at implementing this the other day but getting a margin account open with them looks like a pain
